common/views/components/Contributor/Contributor.js

```diff
@@ -12,53 +13,48 @@ const Contributor = ({
 }: ContributorType) => {
   const descriptionToRender = description || contributor.description;
   const imageProps: ImageProps = contributor.type === 'organisations' ? {
-    width: 64,
-    height: 64,
+    width: 78,
+    height: 78,
```

Comment:
Do we want to be explicitly setting the image height? I'd say an incorrectly aspect-ratio-ed image would looke _more_ broken than a non-square image.

Reply:
I've left it on the contributor, as that's the decision taken by design.
I'll take a look at enforcing that we take a square crop.